openpower/patches/p8dtu-patches/linux/0001-VGA-handshake-with-BMC.patch

```diff
@@ -0,0 +1,67 @@
+From 4c5f6ff7a2b7c2df17d16c3ce18b19036be95223 Mon Sep 17 00:00:00 2001
+From: Jim Yuan <jim.yuan@supermicro.com>
+Date: Mon, 17 Dec 2018 15:57:05 -0800
+Subject: [PATCH] VGA handshake with BMC
```

Comment:
What's the status of this patch going upstream?

Reply:
I am wondering whether this patch can go upstream. It is a ast kernel driver patch only for p8dtu system. Please guide me how to upstream it if it is possible.